src/gen_proposal.jl

```diff
@@ -0,0 +1,46 @@
+function gen(df::AbstractDataFrame, args...; kwargs...)
+	transform(df, args...; kwargs...)
+end
+
+function genby(df::AbstractDataFrame, id, args...; kwargs...)
+	by(df, id, :, args...; kwargs...)
+end
+
+function keep(df::AbstractDataFrame, args...; kwargs...)
+		t = gensym()
```

Comment:
a second small note `gensym()` cannot be used here as it does not guarantee we will not have column name conflicts. But again - as above this is a minor issue than can be fixed.